# Includes UTC time with odometry message.  
# This should stay in sync with odom message by GPS converter.
nav_msgs/Odometry odom
float64 time # UTC time
